CHANGELOG.md

```diff
@@ -610,7 +610,7 @@ Additionally, thanks to [Alexander Turek][@derrabus] for consulting on the repo
 - Include patterns are now ignored when processing STDIN
     - Previously, checks using include patterns were excluded when processing STDIN when no file path was provided via --stdin-path
     - Now, all include and exclude rules are ignored when no file path is provided, allowing all checks to run
-    - If you want include and exclude rules enforced when checking STDIN, use --stdin-path to set the file path
+    - If you want to include and exclude rules enforced when checking STDIN, use --stdin-path to set the file path
```

Comment:
This `to` changes the meaning of the sentence and is incorrect.

Reply:
I just fixed this by amending the original commit.